FastExcel/FastExcel.cs

```diff
@@ -89,6 +89,8 @@ internal void PrepareArchive(bool openSharedStrings = true)
             {
                 SharedStrings = new SharedStrings(Archive);
             }
+
+            loadDefinedNames();
```

Comment:
do you think there is any way we can optionally run this? for speed reasons for users who don't use this feature, maybe optional constructor property or maybe we need a constructor object with settings